.github/workflows/release.yml

```
@@ -19,166 +20,225 @@ concurrency:
-  create-release-pr:
-    name: Create Release PR
+jobs:
+  # Determine if a release is needed
+  check-release:
+    name: Check for Release
     runs-on: ubuntu-latest
-    needs: validate
-    if: github.event_name == 'push' && !contains(github.event.head_commit.message, '[skip ci]')
+    timeout-minutes: 10
+    if: |
+      github.event_name == 'push' && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)